docs/pages/accounts/audit-logs.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Audit logs
+description: A list of changes that happened to selected entities owned by an account.
+---
+
+## What are audit logs?
+
+Our implementation of audit logs will record actions made with Expo services by accounts. Recorded data will include what object was affected, what type of modification was made, who performed the action, and when this activity occurred.
```

Comment:
```suggestion
Audit logs record actions made with Expo Application Services (EAS) by accounts. Recorded data includes information about the affected entities, the type of modification made to them, who performed the action, and when the activity occurred.
```

Let's convey this (and the rest of the section) in a way where the feature is already present (so that the voice usage here is present when we decide to publish these docs along with the feature).